tools/accuracy_checker/requirements.in

```diff
@@ -33,9 +33,6 @@ tokenizers>=0.10.1
 transformers>=4.5
 nltk>=3.5
 
-# editdistance calculation
-editdistance>=0.5.3
```

Comment:
@eaidova Should we synchronize this with openvino-dev ?https://github.com/openvinotoolkit/openvino/blob/576471cc2759db3c511d50b9bd500cfca410edf4/tools/openvino_dev/requirements.txt#L22

Reply:
I provided changes in this PR: https://github.com/openvinotoolkit/openvino/pull/9137